usaspending_api/agencies/management/commands/load_agencies_sql/populate_dependent_tables.sql

```diff
@@ -0,0 +1,119 @@
+delete from cgac;
+delete from frec;
+delete from subtier_agency_new;
+delete from toptier_agency_new;
+
+
+
+insert into
+    cgac (
+        cgac_code,
+        agency_name,
+        agency_abbreviation
+    )
+select
+    cgac_agency_code,
+    max(agency_name),
+    max(agency_abbreviation)
+from
+    temp_raw_agency_csv
+where
+    agency_name is not null and
+    cgac_agency_code is not null
+group by
+    cgac_agency_code;
+
+
+
+insert into
+    frec (
+        frec_code,
+        agency_name,
+        agency_abbreviation
+    )
+select
+    frec,
+    max(frec_entity_description),
+    max(frec_abbreviation)
+from
+    temp_raw_agency_csv
+where
+    frec is not null and
+    frec_entity_description is not null
```

Comment:
So if there's no `frec_entity_description`, we can just not include it in the DB?

Reply:
Since in most cases all we show is the entity name, an entity without a name is meaningless.

Reply:
In terms of errors, it seems like we are choosing between having the search for a frec return nothing when we mess up adding a description, and having a frec with no description be returned. The second seems a little more explicit of a failure to me.